### PR TITLE
Console command (bbapp:create_site) compatibility improvements

### PR DESCRIPTION
**Pull request by gobjila**
Hello, I improved some "exit codes" and "console exceptions" in order to make the command compatible with calls from another console commands or from a web controller.

### CODE REVIEW

File: Console/Command/CreateWebsiteCommand.php [hunk 1]
```diff
@@ -104,12 +105,17 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
+        try{
+            $this
+                ->init($input, $output)
+                ->runSitesYmlProcess()
+                ->runLayoutsGenerationProcess()
+            ;
+        } catch (\Exception $e) {
+            $output->writeln('<error>'.$e->getMessage().'</error>');
+            return 1;
+        }
```
Comment (crouillon) [inline Console/Command/CreateWebsiteCommand.php]: Could you explicitly add a return 0; when everything ok ?

File: Console/Command/CreateWebsiteCommand.php [hunk 2]
```diff
@@ -104,12 +105,17 @@ protected function execute(InputInterface $input, OutputInterface $output)
             throw new \InvalidArgumentException('Both options `site_label` and `site_domain` are required.');
         }
 
-    	$this
-    		->init($input, $output)
-    		->runSitesYmlProcess()
-    		->runLayoutsGenerationProcess()
-		;
-    }        
+        try{
```
Comment (eric-chau) [inline Console/Command/CreateWebsiteCommand.php]: take care about coding style please, a whitespace is missing between `try` and `{`

File: Console/Command/CreateWebsiteCommand.php (hunk 1 above)
Comment (eric-chau) [inline Console/Command/CreateWebsiteCommand.php]: agree with @crouillon

**Comment by crouillon** [comment]
Thanks @gobjila

**Pull request merged**